scripts/cuboid_transformer/earthnet_w_meso/cfg_pretrained.yaml

```diff
@@ -0,0 +1,118 @@
+dataset:
```

Comment:
Can we rename the `cfg_pretrained.yml`? May be  call it `earthformer_enso_v1.yaml`?

Reply:
> Can we rename the `cfg_pretrained.yml`? May be call it `earthformer_earthnet_v1.yaml`?

Done. They are renamed as [`earthformer_sevir_v1.yaml`](https://github.com/gaozhihan/earth-forecasting-transformer/blob/d05bcbb2a983a8424091f46dbbca462813946473/scripts/cuboid_transformer/sevir/earthformer_sevir_v1.yaml), [`earthformer_enso_v1.yaml`](https://github.com/gaozhihan/earth-forecasting-transformer/blob/d05bcbb2a983a8424091f46dbbca462813946473/scripts/cuboid_transformer/enso/earthformer_enso_v1.yaml) and [`earthformer_earthnet_v1.yaml`](https://github.com/gaozhihan/earth-forecasting-transformer/blob/d05bcbb2a983a8424091f46dbbca462813946473/scripts/cuboid_transformer/earthnet_w_meso/earthformer_earthnet_v1.yaml) respectively.